server/util/file.py

```diff
@@ -41,6 +41,7 @@ def s3():  # pylint: disable=invalid-name
         "s3",
         aws_access_key_id=config.AWS_ACCESS_KEY_ID,
         aws_secret_access_key=config.AWS_SECRET_ACCESS_KEY,
+        region_name="us-west-1",
```

Comment:
I had to add this or the url returned in the get presigned post call will not be domain specific which causes errors. I assume this should be an env var?